.github/workflows/pr-sanity.yaml

```diff
@@ -4,6 +4,8 @@ name: Sanity Checks
 
 # Controls when the action will run.
 on:
+  push:
+    branches: [ main ]
```

Comment:
@nunnatsa should not it be "master" ?

Reply:
fixed

Reply:
i would prefer to use native yaml format, but not critical for now.
```
on:
  push:
    branches:
    - master
```